### DISCUSSION TITLE
[Question] Limited Installations

### DISCUSSION THREAD

**Discussion by simonlokk** [arichornlover/YouTubeTweak-Downloads#2] [General]
Hi! I was wondering if you would ever consider  compiling uYouPlus for installation via Cydia on a jailbroken device, rather than just side-loading which has it's own limitations. The version that is available on cydia via  a repo is a bit old at this point. Either way, thanks for your time!

**Comment by arichornlover** [comment]
At the moment I cannot I don’t know what to say about this but at the moment I’m not an advanced iOS user so I cannot do the cynical stuff at this very moment.

**Comment by arichornlover** [comment]
@simonlokk I have done your request! Hope you enjoy!

**Comment by arichornlover** [comment]
I have redesigned the downloads page this time which is no longer limited to sideloading! You can request me more versions of uYouPlus like Cercube later on but hope you like the Jailbreak Download Links!

**Comment by arichornlover** [comment]
and by the way, I highly don’t recommend using jailbreak uYouPlus on the latest version of YouTube because since ichitaso has not updated it in a while, so all I can do is v17.29.3
uYouPlus-For-JB will crash on v17.30.1 or later due to outdated submodules. I’ll tell the dev about this.
